pkgs/applications/networking/giara/default.nix

```diff
@@ -6,17 +6,19 @@
 , ninja
 , python3
 , wrapGAppsHook
-, gtk3
+, gtk4
```

Comment:
Haven't tested yet, but maybe we need to fetchpatch https://gitlab.gnome.org/World/giara/-/commit/6204427f8b8e3d8c72b669717a3f129ffae401d9.patch because the dependency declared in 1.0 seems to be wrong.

Reply:
Will include this patch. Thanks. I tested and it works.